### ISSUE TITLE
To-do List Text Shows Up Above The Tasks 

### ISSUE THREAD

**Issue by akkota** [narlock/OpenLife#5]
![todo list shows up above the tasks](https://user-images.githubusercontent.com/108426332/232353030-8c091004-7099-4328-8d48-96cf0df5a90c.png)

OS: Windows 11 
It fixed when I minimized the screen lol. I'm not sure if this is a real issue but thought I'd include it in here just if it is.

**Comment by narlock** [comment]
Hey, if you are able to describe / reproduce this I can look into solving this.

**Comment by akkota** [comment]
Yeah I'm not sure what happened, I had Kaizen minimized for a bit, like an hour. I opened it up and the to-do list showed up above the tasks. Lemme see if I can reproduce it, but this is the first time it's happened since its release.

**Comment by akkota** [comment]
![error](https://user-images.githubusercontent.com/108426332/233781535-bf42cdf4-bdef-49e7-8bfa-57436cda1b03.png)
It happened again :( 
I minimized Kaizen, opened it about 30 minutes later and it did the same thing. 
(Btw this time the error was on the todo widget)

**Comment by narlock** [comment]
After taking a look at the code, this most likely appears because under an repaint operation to the GUI, the GUI is not be revalidated. Will look into and ensure all lines are revalidating the GUI before closing this issue.
